src/main.rs

```diff
@@ -1200,6 +1204,14 @@ pub const CRC_AUG_CCITT: Algorithm<u16> = Algorithm {
 };
 
 fn program_rp2040() -> io::Result<()> {
+    let bytes = std::fs::read("/etc/cacophony/rp2040-firmware.elf").unwrap(); // Vec<u8>
```

Comment:
Should it panic if the firmware file doesn't exist?  If you expect it to always exist, I think it's better to use `.expect("firmware file should exist at /etc/cacophony/rp2040-firmware.elf")` so we print nicer error messages in the event of a panic.

Reply:
oh yep definately should have that, thanks